lachesis-rs/src/swirlds.rs

```diff
@@ -19,7 +19,9 @@ use std::marker::PhantomData;
 use std::sync::{Arc, Mutex};
 use std::time::{SystemTime, UNIX_EPOCH};
 
-const C: usize = 6;
+// const C is the frequency of coin toss rounds
+// Swirlds paper requires C > 2; let keep it prime
+const C: usize = 3;
```

Comment:
We need to make it consistent with Go-Lachesis.

Reply:
What;s the benefit of it being prime?

Reply:
It's on recommendation from Quan. May be due to some obsession with primes :), though being a prime is not required.